zcash_client_sqlite/src/wallet/transparent.rs

```diff
@@ -866,20 +896,10 @@ pub(crate) fn get_spendable_transparent_outputs<P: consensus::Parameters>(
          FROM transparent_received_outputs u
          JOIN transactions t ON t.id_tx = u.transaction_id
          WHERE u.address = :address
-         AND (
-            -- tx is mined and has at least min_confirmations
-            (
-                t.mined_height < :target_height -- tx is mined
-                AND :target_height - t.mined_height >= :min_confirmations
-            )
-            -- or the utxo can be spent with zero confirmations and is definitely unexpired
-            OR (
-                :min_confirmations = 0
-                AND (t.expiry_height = 0 OR t.expiry_height >= :target_height)
-            )
-         )
-         -- and the output is unspent
-         AND u.id NOT IN ({})",
+         AND u.value_zat > :min_value
```

Comment:
Possible bug: this uses `> MARGINAL_FEE`, but the other two instances use `>= MARGINAL_FEE`. This means the spendable balance may appear larger than what can actually be spent, leaving a user's wallet permanently showing a "Shield funds" button. Given that these notes can always be spent, and sometimes spent without being entirely consumed as fee (using an already-paid-for spend):
```suggestion
         AND u.value_zat >= :min_value
```